### PR TITLE
[ODS-4949] Modify Load Tools Script to Use dotnet CLI.

### PR DESCRIPTION
**Pull request by tnunnerydl**

### CODE REVIEW

File: logistics/scripts/modules/LoadTools.psm1 [hunk 1]
```diff
@@ -26,31 +26,43 @@ function Invoke-BuildLoadTools {
...
+     if (!(Test-Path -LiteralPath $buildLogFilePath -PathType Leaf)) {
+        Write-Warning ("Cannot find the build log file at '$buildLogFilePath', so unable to determine if build succeeded or not.")
+        return
     }
+
+    # Get if the build succeeded or not.
+    [bool] $buildFailed = (Select-String -Path $buildLogFilePath -Pattern "Build FAILED." -SimpleMatch -Quiet)
+
+    if ($buildFailed) {
+        Write-Host 'Ensure that the Visual Studio SDK is installed.'
```
Comment (tbmeador-dlp) [inline logistics/scripts/modules/LoadTools.psm1:61]: can we remove this? it doesn't make sense here.
```suggestion
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
Looks good overall. Does it work if we remove the `Invoke-RestoreLoadToolsPackages` function? They should be restored automatically by the `dotnet build` in the `Invoke-BuildLoadTools` function.

**Comment by tnunnerydl** [comment]
It worked after removing Invoke-RestoreLoadToolsPackages.

File: logistics/scripts/modules/LoadTools.psm1 [hunk 2]
```diff
@@ -11,46 +11,48 @@ Import-Module -Force -Scope Global (Get-RepositoryResolvedPath "logistics\script
...
 function Invoke-BuildLoadTools {
     param(
         [hashtable] $config
     )
 
     $buildConfiguration = $config.buildConfiguration
+    $solutionPath = $config.loadToolsSolution
+    $verbosity = "minimal"
+
+    if ((Get-DeploymentSettings).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
```
Comment (tbmeador-dlp) [inline logistics/scripts/modules/LoadTools.psm1:23]: This line seems to be causing the template builds to fail. We don't have a `Npgsql` build configuration anymore so we should be able to remove it.
```suggestion
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**